src/main/java/frc/robot/subsystems/swervev3/SwerveDrivetrain.java

```diff
@@ -246,6 +257,29 @@ public boolean isFacingTarget() {
     return facingTarget;
   }
 
+  private static final DCMotor driveMotor = DCMotor.getNEO(1);
```

Comment:
All simulation related code should be moved to a separate class class and then created and managed by a `SimDrivetrainIO`